Makefile

```diff
@@ -107,6 +110,51 @@ $(CONTROLLER_GEN):
 .PHONY: controller-gen
 controller-gen: $(CONTROLLER_GEN)
 
+.PHONY: cluster/prepare/local
+cluster/prepare/local: kustomize cluster/prepare/project install cluster/create/system-redis cluster/create/backend-redis cluster/create/provision-database
```

Comment:
@MStokluska looks very nice, but seems missing "make download" step, need add download target here also?

Reply:
go mod download is not necessary. I never run it.